lib/modules/manager/npm/post-update/npm.ts

```diff
@@ -48,7 +82,9 @@ export async function generateLockFile(
       toolName: 'npm',
       constraint:
         config.constraints?.npm ??
-        getPackageManagerVersion('npm', await lazyPgkJson.getValue()),
+        getPackageManagerVersion('npm', await lazyPgkJson.getValue()) ??
```

Comment:
While looking into this myself I noticed a slight typo(?). I think it should be `lazyPkgJson`, correct?